apps/files/src/components/LocationPicker/LocationPicker.vue

```diff
@@ -306,10 +316,17 @@ export default {
       this.$router.push({ name: 'files-list', params: { item: target || '/ ' } })
     },
 
-    async moveResources() {
+    isRowDisabled(resource) {
+      const isBeingMoved = this.resources.some(item => item === resource.path)
+
+      return resource.type !== 'folder' || !resource.canCreate() || isBeingMoved
```

Comment:
nice, so your are indeed checking the perm on the target folder :+1:

Reply:
would be good to add a test for the case(s) where a target folder is not targettable due to missing permissions (received read-only share)

Reply:
I've opened a ticket for collecting the next scenarios to test for copy and move. Should I add this in there or implement it already in this PR?

Reply:
later is fine, please label the ticket "QA-team" and ping Artur

Reply:
later... but within the sprint would be good because else I don't think we can consider the task "done"